Templates/index.md

```diff
@@ -0,0 +1,32 @@
+# How to Use EOS Doc Templates
+
+To submit your documentation to the EOS Network Foundation (ENF), create a fork of the receiving repository and then add work on your own fork. From there, do a PR from your fork to the main branch. DevRel then reviews the PR, suggests changes, if any, and after approval the PR can be merged. 
+
+ENF Templates use MarkDown language.
+
+Templates provide a structure for your content.
+
+Text identified within brackets [...] are instructions to you for the type of information to include in the topic.
+
+If a section title contains "(Optional)" and you don’t need to add information for that topic, add the description “N/A”.
+
+Please include your email or telegram address so DevRel can reach out to you with any questions.
+
```

Comment:
Since this file should also act as a index to the contents inside the `Templates` folder, you should also add a new section here called "Available Templates", a stem sentence, and the list of templates provided with a link to each. This will also simplify the next section which is a bit crowded already.